support/mock/mockrt.go

```diff
@@ -585,6 +585,10 @@ func (rt *Runtime) GetState(o runtime.CBORUnmarshaler) {
 	}
 }
 
+func (rt *Runtime) SetStateForTesting(o runtime.CBORMarshaler) {
```

Comment:
Move this method into the "mocking facilities" section. 

Since it's a mock the `ForTesting` is probably implicit. Maybe `ReplaceState`? or `OverwriteState`?